### PR TITLE
Fixing figure6

### PR DESCRIPTION
**Pull request by andrewram4287**

### CODE REVIEW

**Review by aarmey — CHANGES_REQUESTED** [review]
I'm confused. What is this fixing?

**Comment by andrewram4287** [comment]
> I'm confused. What is this fixing?

Just changing the naming of the clusters. It started at 0 but now its starting at 1.

**Comment by aarmey** [comment]
Ok. There are other edits here besides that. Should those be removed?

File: gmm/figures/figure10.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def makeFigure():
     ]
     maximizedFactors[2] = reorder_table(maximizedFactors[2])
 
-    NK_DF = pd.DataFrame(data=maximizedNK, columns=clustArray, index=rankCol)
+    NK_DF = pd.DataFrame(data=maximizedNK, columns=rankCol, index=clustArray)
```
Comment (andrewram4287) [inline gmm/figures/figure10.py:45]: @aarmey This is the same change as the main so it should be fine!

**Comment by aarmey** [comment]
Merging!

**Pull request merged**